4-HTTP/server/index.js

```diff
@@ -0,0 +1,38 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const port = 3000;
+
+app.use(cors());
+app.use(
+  express.urlencoded({
+    extended: true,
+  })
+);
+app.use(express.json());
+
+let list = [];
+
+app.get("/", (req, res) => {
+  res.send(list);
+});
+
+app.post("/", (req, res) => {
+  list.push(req.body);
+  res.send(list);
+});
```

Comment:
It's not a clear API to send a post request to the root path...
I would give it a name: `app.post("/setTodos")` or something

Reply:
Alright, so I changed all the server routes to "/todo" and for get all todos its "/todos", seems clearer.